src/test/resources/mapper/core/role_share_resource_limit/map_share_resource_limits/deployments.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "name": "deployment1",
+    "defaultRoleShareResourceLimit": {
+      "maxAcceptedUsers": 10,
+      "invitationTtl": null
+    }
+  },
+  {
+    "name": "deployment2",
+    "defaultRoleShareResourceLimit": {
+      "maxAcceptedUsers": 10,
+      "invitationTtl": null
+    }
+  },
+  {
+    "name": "deployment3",
+    "defaultRoleShareResourceLimit": {
+      "maxAcceptedUsers": 10,
+      "invitationTtl": null
```

Comment:
remove invitationTtl field for some of the default resource limits. to test withdefault value